File: frontend/src/types/answer.ts
/**
 * 정답/오답 관련 타입 정의
 */
export interface Answer {
  answer_id: string;
  saved: boolean;
}

export interface AnswerCreate {
  user_id: string;
  unit_id: string;
  selected: number;
  is_correct: boolean;
}

File: backend/app/schemas/answer.py
"""
정답/오답 관련 Pydantic 스키마
"""
from pydantic import BaseModel
from typing import Optional


class AnswerCreate(BaseModel):
    user_id: str
    unit_id: str
    selected: int
    is_correct: bool


class AnswerResponse(BaseModel):
    answer_id: str
    saved: bool
